src/js/d3/sunburst.js

```diff
@@ -0,0 +1,194 @@
+import logger from '../logger';
+import * as d3 from 'd3';
+
+
+export default (options) => {
+  const {
+    width,
+    height,
+    data,
+  } = options;
+
+  var radius = Math.min(width, height) / 2;
```

Comment:
Any chance we could ES6 this? Seems kind of odd that it's an arrow function with a bunch of ES5 here